contracts/NFT.vy

```diff
@@ -132,7 +132,7 @@ def __init__(activeUserAddress: address, _password: uint256):  # activeUserContr
     self.name = "Not-So-Fungible Wolvies"
     self.symbol = "NSFW"
     self.minter = msg.sender
-    self.baseURI = "http://ipfs.wolvercoin.com/ipfs/"
+    self.baseURI = ""
```

Comment:
...??  Taking out the BaseURI eh

Reply:
IPFS was not online during the time I was testing the app, so I had to delete the prefix so I could use arbitrary imgur URLs because I could not upload to IPFS. We can revert this change if we turn back on IPFS